generator.yaml

```diff
@@ -540,6 +540,8 @@ resources:
       Tags:
         compare:
           is_ignored: true
+    tags:
```

Comment:
@vijtrip2 could you describe what this means? since we already set ignore for comparing Tags. is this a breaking change?

Reply:
No, it is not a breaking change. This configuration will just not add ACK system tags for this resource.
When I was fixing the e2e tests, i saw the following error message, and ignored setting ACK system tags for this resource.

```
Tags are not supported in Model Package versions. Please add them to the Model Package Group.                                                                                                    │
│                            status code: 400, request id: ee23b541-c579-4f9e-9bc1-8ef1b081112e
```